edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/RegistryController.cs

```diff
@@ -212,7 +212,7 @@ public class RegistryController : Controller
 
                 if (!await this.AuthenticateAsync(deviceId, Option.None<string>(), Option.None<string>()))
                 {
-                    await this.SendResponseAsync(HttpStatusCode.Unauthorized);
+                    await this.SendResponseAsync(HttpStatusCode.BadRequest);
```

Comment:
Is this correct? If the entire request is correct but the token is invalid, IoTHub returns 400?

Reply:
What I did here was wrong. I had 2 PR. Forgot to put this on the master master PR.